### PR TITLE
Add Prayer Menu Tests

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR just adds a test for the `PrayerMenu` and `PrayerMenuCard` components because I didn't add them originally. 

### How do I test this PR?
Just look at the tests and the snapshots and make sure that they look good.
Run `yarn test` to make sure they don't break any other tests.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [N/A] Closes [DEV-XXX](#url)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by redreceipt — APPROVED** [review]

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap [hunk 1]
```diff
@@ -0,0 +1,1034 @@
...
+    </View>
+    <View
+      collapsable={false}
+      style={
+        Array [
+          Object {
+            "flex": 1,
+            "overflow": "hidden",
+          },
+          undefined,
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:66]: Is this undefined intended?

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap [hunk 2]
```diff
@@ -0,0 +1,1034 @@
...
+      >
+        <View>
+          <View
+            onLayout={[Function]}
+            style={
+              Array [
+                Object {
+                  "flexDirection": "row",
+                },
+                null,
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:187]: Is this `null` intended?

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap [hunk 3]
```diff
@@ -0,0 +1,1034 @@
...
+                      }
+                    >
+                      <Image
+                        onLoad={[Function]}
+                        source={
+                          Array [
+                            Object {
+                              "cache": "force-cache",
+                              "uri": "https://picsum.photos/600/400/?random",
+                              "url": "https://picsum.photos/600/400/?random",
```
Comment (IsaacHardy) [inline packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:258]: A url and an uri?

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
Had a few questiosn about the snapshot

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap (hunk 3 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:258]: I see this in TAP snapshots so I'm guessing it's okay.

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap (hunk 1 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:66]: I think this is when the `<TabView>` starts to be rendered. The only place in TAP that I can find this in the the devotional and it does NOT have `undefined` there. We extracted out and are using the React Native `<TabView>`. In TAP, there is a line that wraps the TabView with a `flex: 1` (const withStyles = styled({ flex: 1 }, 'TabView');). I think that's the difference ... so I think this is okay.

File: packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap (hunk 2 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap:187]: According to the tests in TAP on the discover tab, this `null` is also fine. Whew!

**Review by IsaacHardy — APPROVED** [review]

**Pull request merged**